Resources/translations/FOSUserBundle.hu.yml

```diff
@@ -61,19 +69,20 @@ resetting:
             A jelszó lecseréléséhez kérjük látogasson el a következő linkre: %confirmationUrl%
 
             Üdvözlettel,
-            the Team.
+            a Csapat.
 
 layout:
     logout: Kijelentkezés
     login: Bejelentkezés
     register: Regisztráció
     logged_in_as: 'Belépve mint %username%'
+
 form:
     group_name: 'Csoport név'
     username: Felhasználónév
     email: E-mail
     current_password: 'Jelenlegi jelszó'
     password: Jelszó
-    password_confirmation: Megerősítés
+    password_confirmation: 'Jelszó megerősítése'
     new_password: 'Új jelszó'
-    new_password_confirmation: Megerősítés
+    new_password_confirmation: 'Új jelszó megerősítése'
```

Comment:
Felőlem maradhat is, mi mondjuk a "Jelszó újra" verziót szoktuk használni, de a megerősítés sem rossz.